app/controllers/stores/sessions_controller.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class Stores::SessionsController < Devise::SessionsController
+  # before_action :configure_sign_in_params, only: [:create]
+  def after_sign_in_path_for(resource)
```

Comment:
Unused method argument - resource. If it's necessary, use _ or _resource as an argument name to indicate that it won't be used. You can also write as after_sign_in_path_for(*) if you want the method to accept any arguments but don't care about them.